tests/backend/test_github_integration_extended.py

```
@@ -0,0 +1,298 @@
+import pytest
+from unittest.mock import patch, Mock
+from src.backend.github_integration import GitHubIntegration
+from src.backend.exceptions import GitHubAuthError, WebhookError
```

Ruff (0.8.2)

`src.backend.exceptions.GitHubAuthError` imported but unused

Remove unused import: `src.backend.exceptions.GitHubAuthError`

(F401)